lib/router/router.js

```diff
@@ -109,6 +126,15 @@ p.getInfo = function(req) {
     }
   }
 
+  // forward original image if no operation on image and browser support image format
+  if (routeInfo.imageSteps.length === 0 && this.options.ifNoOperation && this.options.ifNoOperation.justForward === true) {
```

Comment:
With the new option, what's the point of using isteam if you don't ever intend to process the images? Sounds like all you need is a proxy. I get wanting to do this via optional query ^ as described above.